### ISSUE TITLE
训练做loss真值问题，才疏学浅，如有不妥，望请赐教

### ISSUE THREAD

**Issue by jack-wang233** [windygoo/PromptNucSeg#13]
在dataset.py文件中116-129行中赋值给inst_map ，这个是选择出来的部分分割图像，不是全部真值图像，做loss要用这个部分分割，而不是全部分割图像吗？如有不妥，不吝指教 ...
             for pid in chosen_pids:
                   mask_single_cell = torch.eq(inst_map, pid)

                   inst_maps.append(mask_single_cell)
                   prompt_points.append(
                       random.choice(
                           torch.argwhere(mask_single_cell)
                    )[None, [1, 0]].float())

            prompt_points = torch.stack(prompt_points, dim=0)
            prompt_labels = torch.ones(prompt_points.shape[:2])
            cell_types = torch.as_tensor(cell_types)

            inst_map = torch.stack(inst_maps, dim=0)

**Comment by windygoo** [comment]
是的。输入一个point prompt，目标是分割这个prompt对应的细胞，而不是全部的细胞。
